test/MOIWrapper.jl

```diff
@@ -0,0 +1,36 @@
+using CPLEX, Base.Test, MathOptInterface, MathOptInterface.Test
```

Comment:
You should test this on v0.7 as well. It will need `Compat.Test` instead of `Base.Test`, or we can leave that for another PR.